spec/rails_helper.rb

```diff
@@ -1,62 +1,25 @@
-# This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV["RAILS_ENV"] ||= 'test'
-require 'simplecov'
-require 'coveralls'
-Coveralls.wear!('rails')
-SimpleCov.start('rails')
+require 'spec_helper'
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'missing_translations_helper'
-
 require 'factory_girl'
 
-
+Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |file| require file }
```

Comment:
this line (including `capybara.rb`) is why I got into making all these changes in this file -- sorry for the diff noise! Can separate it out if too distracting